CMakeLists.txt

```diff
@@ -696,10 +719,22 @@ if(CP2K_USE_SIRIUS)
   find_package(sirius REQUIRED)
 endif()
 
+if(CP2K_USE_METIS)
```

Comment:
Could you please remove all mentions of METIS (there are a few more below). I just removed METIS in #3875.

Otherwise this PR looks good to me. Thanks for going the extra mile and patching libgrpp.